### ISSUE TITLE
Owncloud context menu options differ when file has not been synced yet

### ISSUE THREAD

**Issue by HanaGemela** [owncloud/client#7330]
Client: 2.6.0alpha2 (build 12128)
macOS: 10.14.5
Server: 10.2.1 (stable)

Steps to recreate:
1) Create a new file 
2) Quickly open the ownCloud context menu by right clicking on the file before it gets synced

Actual result: Delete and Move and Upload options are available but once the file is synced they disappear. Is there any reason why the option to delete or move is dependant on the state of the file if it has been synced or not? 

Expected result: Not really sure. Should the options be always available or are they displayed by mistake?

![image](https://user-images.githubusercontent.com/49001702/61527070-f0111100-aa1b-11e9-94cc-1e691bf76f5a.png)

**Comment by jnweiger** [comment]
References:
* https://github.com/owncloud/client/issues/6252
* https://github.com/owncloud/core/issues/29873

**Comment by jnweiger** [comment]
The menu entry 'Move and upload...' also appears in some other possibly unwanted situations:
```
Misfeature: this should not happen, imho

Scenario: Folder excluded from selective sync receives a new file although it is readonly
  GIVEN folder "foobar" was fully synced 
  AND user changes the folder to readonly e.g. via `chmod a-w foobar`
  AND user disables the folder via selective sync. 
  WHEN root user creates a file "foobar/newfile.txt"
  THEN the clients browser integration shows a yellow exclamation mark overlay icon on that file
  AND the ownCloud context menu has an extra entry "Move and upload..."
```

**Comment by ogoffart** [comment]
PR: https://github.com/owncloud/client/pull/7527

**Comment by ogoffart** [comment]
The PR should remove these menu entry, that were there by mistake as the code thought this was a read-only folder while it wasn't

**Comment by HanaGemela** [comment]
Fixed in 2.6.1-daily20191127 (build 12791), macOS 10.15.1
Still broken in 2.6.1daily20191127 (build 12732), Windows 10

**Comment by HanaGemela** [comment]
Fixed in 2.6.1sprint2 (build 12978), macOS 10.15.2
